tests/debugger/bug01656.phpt

```diff
@@ -8,12 +8,14 @@ xdebug.start_with_request=yes
 xdebug.client_discovery_header=I_LIKE_COOKIES
 xdebug.discover_client_host=1
 xdebug.client_port=9999
-xdebug.log=
+xdebug.log={TMP}/{RUNID}{TEST_PHP_WORKER}bug01656.txt
```

Comment:
Given that now we are not printing the failed connection attempt in the std output, we need to read it from the log to confirm that it contains the expected data